examples/compat/ggplot/density.py

```diff
@@ -9,4 +9,4 @@
 
 plt.title("Density ggplot-based plot in Bokeh.")
 
-mpl.to_bokeh(name="density")
+mpl.show(mpl.to_bokeh(name="density"))
```

Comment:
If you delete the `show` import at line https://github.com/bokeh/bokeh/pull/1634/files#diff-90fe78313584b02450b1fb1407f58207R31, the Python's "magic" import behaviour disappear...
If I have to choose, I would like to see the `show` imported directly from `plotting` as I said before.